npc/quests/quests_nameless.txt

```diff
@@ -61,8 +61,9 @@
 //= 2.4
 //=========================================================================
 
-//== Nameless Island and Cursed Monestary Access Quest :: aru_monas =
-airplane_01,90,63,0	script	Event Switch#pc	FAKE_NPC,3,3,{
+// Nameless Island and Cursed Monestary Access Quest :: aru_monas
+//=========================================================================
+airplane_01,90,63,0	script	Event Switch#pc	-1,3,3,{
```

Comment:
here should be FAKE_NPC and not -1

Reply:
done, kindly help re-review the commits too :D